bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/State.java

```diff
@@ -184,4 +189,51 @@ public Effect getEffect() {
     public boolean isReachable() {
         return reachable;
     }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see java.lang.Object#equals(java.lang.Object)
+     */
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Java has a standard contract for equals/hashcode that two equal objects must have the same hashcode. So just generate a hashcode method for this class and I think we can call this PR done.

Reply:
Added hashcode and removed Javadoc